pallets/compliance-manager/src/lib.rs

```diff
@@ -703,7 +799,19 @@ impl<T: Trait> Module<T> {
         primary_issuance_agent: Option<IdentityId>,
     ) -> AssetComplianceResult {
         let asset_compliance = Self::asset_compliance(ticker);
+        // To know whether the given ticker has the implicit requirements active or not.
+        // if `yes` then it returns the `Some(ConditionResult)` for the `from_did_opt` and `to_did_opt`.
+        // if `no` then it returns None for both.
+        let (from_result, to_result) = Self::get_implicit_condition_result(
+            ticker,
+            from_did_opt,
+            to_did_opt,
+            primary_issuance_agent,
+        );
         let mut asset_compliance_with_results = AssetComplianceResult::from(asset_compliance);
+        // `implicit_requirements` by default is `None` here but we re-compute them as per the above results.
+        asset_compliance_with_results.implicit_requirements =
+            AssetComplianceResult::get_implicit_requirements(from_result, to_result);
```

Comment:
This isn't required. We can directly return `from_result` and `from_result` in the rpc.